pkg/armclient/armclient.go

```diff
@@ -33,12 +33,15 @@ type Client struct {
 
 // LegacyInstance is a singleton ARMClient used while migrating to the
 // injected client
-var LegacyInstance = NewClientFromCLI()
+var LegacyInstance Client
 
 // NewClientFromCLI creates a new client using the auth details on disk used by the azurecli
-func NewClientFromCLI() *Client {
+func NewClientFromCLI(subscription string) *Client {
```

Comment:
Nit: this looks like it takes in a `tenantID` so param should this be `tenantID` or have I misunderstood

Reply:
Darn - hangover from initial spike on this!